.github/workflows/prettier.yml

```
@@ -0,0 +1,20 @@
+name: 'Format with Prettier'
+on:
+  pull_request:
+      branches: [master]
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
```

I think the latest version without the `with` should also work here:

```suggestion
        uses: actions/checkout@v4
```